package.json

```diff
@@ -20,7 +20,7 @@
     "@mapbox/glyph-pbf-composite": "0.0.3",
     "@mapbox/mapbox-gl-native": "5.0.2",
     "@mapbox/mapbox-gl-style-spec": "13.12.0",
-    "@mapbox/mbtiles": "0.11.0",
+    "@mapbox/mbtiles": "0.12.1",
```

Comment:
This causes breaking issue as v0.12.x has dropped support for Node 10.

https://github.com/mapbox/node-mbtiles/blob/master/CHANGELOG.md#0120

Reply:
I mean, Node10 security support ended over a year ago, we really shouldn't be basing changes and updates (especially to vulnerability-related items) on past EOL Node versions...

Reply:
This should be tested on 14,16,18 based on current support lifecycle.